src/CMakeLists.txt

```
@@ -573,6 +574,23 @@ if(NOT ALICEVISION_USE_USD STREQUAL "OFF")
   endif()
 endif()
 
+# ==============================================================================
+# GeographicLib
+# ==============================================================================
+# - optional, used to convert to UTM coordinate system in SfmTransform
+# ==============================================================================
+set(ALICEVISION_HAVE_GEOGRAPHIC 0)
+
+if(NOT ALICEVISION_USE_GEOGRAPHIC STREQUAL "OFF")
+  find_library(GeographicLib_LIBRARY NAMES Geographic GeographicLib DOC "GeographicLib library")
```

```suggestion
  find_package(GeographicLib CONFIG)
```
then the target `GeographicLib::GeographicLib_SHARED` can be used to express the dependency in the remaining cmake parts